udata/tests/test_discussions.py

```diff
@@ -100,6 +100,40 @@ def test_new_discussion_missing_subject(self):
         })
         self.assertStatus(response, 400)
 
+    def test_new_discussion_with_extras(self):
+        user = self.login()
+        dataset = Dataset.objects.create(title='Test dataset',
+                                         extras={'key': 'value'})
+
+        with self.assert_emit(on_new_discussion):
+            response = self.post(url_for('api.discussions'), {
+                'title': 'test title',
+                'comment': 'bla bla',
+                'subject': {
+                    'class': 'Dataset',
+                    'id': dataset.id,
+                },
+                'extras': {'key': 'value'}
+            })
+        self.assert201(response)
+
+        discussions = Discussion.objects(subject=dataset)
+        self.assertEqual(len(discussions), 1)
+
+        discussion = discussions[0]
+        self.assertEqual(discussion.user, user)
+        self.assertEqual(len(discussion.discussion), 1)
+        self.assertIsNotNone(discussion.created)
+        self.assertIsNone(discussion.closed)
```

Comment:
Why is it important to check that `discussion.created` and `discussion.closed` aren't in the discussion ?

Reply:
It's a copy/paste from the add_new_discussion test.
It could be deleted.